src/Pgsql/PgsqlDriver.php

```diff
@@ -221,6 +221,29 @@ public function getTableCreate($tables)
         return [];
     }
 
+    /**
+     * Parses the fully qualified or bare name of the database object (table, view, procedure etc.) into object
+     * with fields "schema" and "table" respectively
+     *
+     * @param string $table Name of the database object with schema or without it
+     *
+     * @return object
+     */
+    private function parseIdent($table)
+    {
+        $query = $this->getQuery(1);
+        $query->select(['r[1]','r[2]'])
+        ->from("parse_ident(:t) as r")
+        ->bind(':t', $table);
+        $qn = $this->setQuery($query)
+        ->loadRow();
+        if ($qn[1]) {
+            return (object) ['schema' => $qn[0], 'table' => $qn[1]];
+        } else {
+            return (object) ['schema' => $this->getDefaultSchema(), 'table' => $qn[0]];
+        }
```

Comment:
```suggestion
        }

        return (object) ['schema' => $this->getDefaultSchema(), 'table' => $qn[0]];
```
No need for `else` if `if` code ends with return.